src/common/modules/navigations/gnb/modules/gnb-recent-favorite/modules/GNBFavorite.vue

```diff
@@ -314,27 +318,22 @@ export default {
             padding-bottom: 0.5rem;
         }
     }
-    .no-data {
-        text-align: center;
-        padding: 3rem 3.25rem;
-        .img {
-            margin: auto;
-        }
-        .text {
-            @apply text-gray-400;
-            font-size: 0.875rem;
-            line-height: 1.5;
-            padding-top: 1.5rem;
-        }
-        .button-wrapper {
-            display: flex;
-            gap: 0.5rem;
-            justify-content: center;
-            padding-top: 1rem;
-            .p-button {
-                width: 10.5rem;
-            }
-        }
+}
+
+/* custom design-system component - p-empty */
+:deep(.p-empty) {
+    text-align: center;
+    padding: 3rem 3.25rem;
+    .text {
+        @apply text-gray-400;
+        font-size: 0.875rem;
+        line-height: 1.5;
```

Comment:
```suggestion
        @apply text-paragraph-md;
```

please refer to [this guide](https://storybook.developer.spaceone.dev/?path=/docs/foundation-styles-typography--font-size)